HesoyamHospital/GraphicEditor/View/Search.xaml.cs

```diff
@@ -24,9 +25,27 @@ public Search()
         private void Search_Button_Click(object sender, RoutedEventArgs e)
         {
             String name = objectName.Text;
-            SearchService search_service = new SearchService();
-            List<MapLocation> results = search_service.Find_objects_by_name(name);
-            dataGridSearch.ItemsSource = results;
+            SearchService searchService = new SearchService();
+
+            if (searchType.SelectedIndex == 2)
+            {
+                if (name.IsNullOrEmpty())
+                {
+                    List<EquipmentAndMedicine> results = searchService.FindAllEquipmentAndMedicines("equipmentandmedicine.txt");
```

Comment:
Equipment and medicine should be placed within a database. If you are not familiar with how adding a new entity to database work, contact a member from Devops team or Web team to get introduced to how ORM works.